src/phone-input/styled-components.js

```diff
@@ -15,22 +13,23 @@ import {
   StyledRoot as SelectStyledRoot,
 } from '../select/index.js';
 import defaultProps from '../select/default-props.js';
+import type {SizeT} from './types.js';
 
 type SizeStyleProps = {
-  $size?: $Keys<typeof SIZE>,
+  $size?: SizeT,
 };
 type HeightStyleProps = {$height: string};
 
-export const StyledFlag = styled<typeof Flag, SizeStyleProps>(
```

Comment:
`StyledFlag` export needs to be maintained

Reply:
otherwise, lgtm

Reply:
I moved it to the `flag.js` and still export from `index`